pkg/csi/service/vanilla/controller.go

```diff
@@ -1725,23 +1725,15 @@ func (c *controller) createFileVolume(ctx context.Context, req *csi.CreateVolume
 				return nil, csifault.CSIInvalidArgumentFault, logger.LogNewErrorCode(log, codes.InvalidArgument,
 					"accessibility requirements cannot be nil for a multi-VC environment")
 			}
-			// Get the accessibility requirements according to the VC they belong to.
+		}
+		if topologyRequirement != nil {
+			// Get accessibility requirements.
 			vcTopologySegmentsMap, err = common.GetAccessibilityRequirementsByVC(ctx, topologyRequirement)
 			if err != nil {
 				return nil, csifault.CSIInternalFault, logger.LogNewErrorCodef(log, codes.Internal,
 					"failed to get accessibility requirements by VC. Error: %+v", err)
 			}
-			log.Debugf("Topology accessibility requirements per VC are %+v", vcTopologySegmentsMap)
-		} else {
-			if topologyRequirement != nil {
-				// Get accessibility requirements.
-				for _, topology := range topologyRequirement.Preferred {
-					vcTopologySegmentsMap[c.managers.CnsConfig.Global.VCenterIP] = append(
-						vcTopologySegmentsMap[c.managers.CnsConfig.Global.VCenterIP],
-						topology.GetSegments())
-				}
-				log.Debugf("Topology accessibility requirements per VC are %+v", vcTopologySegmentsMap)
-			}
+			log.Debugf("Topology accessibility requirements for VC are %+v", vcTopologySegmentsMap)
```

Comment:
Would it be better if this an Info log?
Also nit: `accessibility requirements for VC` -> `accessibility requirements per VC`